.github/workflows/ci.yml

```diff
@@ -95,7 +94,6 @@ jobs:
           - helia-jest-typescript
           - helia-nestjs
           - helia-parcel
-          - helia-remote-pinning
```

Comment:
must have been a merge gone wrong, we shouldn't remove this

Reply:
okay